v2/reputation/types.go

```diff
@@ -297,11 +297,28 @@ func (x *SendLocalTrustResponse) SetBody(v *SendLocalTrustResponseBody) {
 
 // SendIntermediateResultRequestBody is a structure of SendIntermediateResult request body.
 type SendIntermediateResultRequestBody struct {
-	iter uint32
+	epoch uint64
```

Comment:
One more empty line there to keep code style the same.

Reply:
added